extensions/azurecore/src/azurecore.d.ts

```diff
@@ -75,10 +82,18 @@ declare module 'azurecore' {
 		getStorageAccounts(account: azdata.Account, subscriptions: azureResource.AzureResourceSubscription[], ignoreErrors?: boolean): Promise<GetStorageAccountResult>;
 		getBlobContainers(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, storageAccount: azureResource.AzureGraphResource, ignoreErrors?: boolean): Promise<GetBlobContainersResult>;
 		getFileShares(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, storageAccount: azureResource.AzureGraphResource, ignoreErrors?: boolean): Promise<GetFileSharesResult>;
-		getMigrationController(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, resourceGroupName: string, regionName: string, controllerName: string, ignoreErrors?: boolean): Promise<GetMigrationControllerResult>;
-		createMigrationController(account:azdata.Account, subscription: azureResource.AzureResourceSubscription, resourceGroupName: string, regionName: string, controllerName: string, ignoreErrors?:boolean): Promise<CreateMigrationControllerResult>;
-		getMigrationControllerAuthKeys(account: azdata.Account, subscription: azureResource.AzureResourceSubscription, resourceGroupName: string, regionName: string, controllerName: string, ignoreErrors?: boolean): Promise<GetMigrationControllerAuthKeysResult>;
-
+		/**
+		 * Makes Azure REST requests to create, retireve, update or delete access to azure service's resources.
+		 * For reference to different service URLs, See https://docs.microsoft.com/en-us/rest/api/?view=Azure
+		 * @param account The azure account used to acquire access token
+		 * @param subscription The subscription under azure account where the service will perform operations.
+		 * @param serviceUrl The endpoint for the service. See https://docs.microsoft.com/en-us/rest/api/?view=Azure
```

Comment:
serviceUrl doesn't really make sense - this isn't normally a URL. Something like `requestPath` would be better